include/blkdev.h

```diff
@@ -64,6 +64,11 @@
 #  define BLKDISCARDZEROES _IO(0x12,124)
 # endif
 
+/* discard zeroes support, introduced in 5.14 (commit 7957d93b) */
```

Comment:
`/* get disk sequence number, introduced in 5.15 ... */`

The commit is right...

Reply:
Good catch. This commit is actually from #1979. I'll fix it there, too.